packages/cli/src/constructs/check-group.ts

```diff
@@ -189,14 +191,21 @@ export class CheckGroup extends Construct {
     this.browserChecks = props.browserChecks
     const fileAbsolutePath = Session.checkFileAbsolutePath!
     if (props.browserChecks?.testMatch) {
-      this.__addChecks(fileAbsolutePath, props.browserChecks.testMatch)
+      this.__addChecks(fileAbsolutePath, props.browserChecks.testMatch, CheckTypes.BROWSER)
+    }
+    if (props.multiStepChecks?.testMatch) {
```

Comment:
Looking at that PR I also noticed that we're using the check group's browser check defaults as multistep check defaults. I think it makes sense to fix that: https://github.com/checkly/checkly-cli/pull/868/files#diff-5679022c8ca8164d11865b9bd6417b09260e3c4bbbf9386c468c3f53af16ccd8R250-R254